src/parselmouth/PointProcess_docstrings.h

```diff
@@ -0,0 +1,896 @@
+namespace parselmouth
+{
+
+constexpr auto CREATE_CLASS_DOCSTRING = R"(Praat PointProcess.
+
+A sequence object contain a sequence of points $t_i$ in time, defined
+on a domain [`xmin`, `xmax`]. The points are sorted in time, i.e., 
+$t_i+1 > t_i$.
+
+Attributes
+----------
+tmin : float, readonly
+    Starting time of the analysis domain in seconds
+tmax : float, readonly
+    Ending time of the analysis domain in seconds
+
+Methods
+-------
+get_number_of_points()
+    Get the number of time points
+
+get_number_of_periods()
+    Get the number of periods
+
+get_count_and_fraction_of_voice_breaks(from_time=0.0, to_time=0.0, 
+                                       period_ceiling=0.02)
+    Get voice break analysis outputs
+
+get_mean_period(from_time=0.0, to_time=0.0, period_floor=0.0001, 
+                period_ceiling=0.02, maximum_period_factor=1.3)
+    Get average of voice periods
+
+get_stdev_period(from_time=0.0, to_time=0.0, period_floor=0.0001, 
+                period_ceiling=0.02, maximum_period_factor=1.3)
+    Get standard deviation of voice periods
```

Comment:
I'm ... very hesitant about this, mainly because `help` and Sphinx generate this automatically based on the actual methods, and because it puts an immense expectation/standard on other classes and future development. And the same kind of goes for the suggestions your IDE will give when programming.
So why exactly did you add this?

Reply:
(At the same time, I also feel really bad saying that, because you clearly did an immense effort to get this here! And it's kind of hard to argue against more documentation, _especially_ once it's already there :-/ )

Reply:
> So why exactly did you add this?

Because documentation tips I've seen around said to add them... But I did see a thread somewhere on whether to have Methods section or not for class documentations. I suppose the auto-generated method listing would take the signature and the first line of the method documentation. So that should be OK.

Again, feel free to delete whatever you feel is extraneous. As long as you never ask me to redo what you deleted, I take no issue with it.